src/backend/expungeservice/endpoints/auth.py

```diff
@@ -76,29 +77,29 @@ def authorized(f, admin_required, *args, **kwargs):
             jwt.exceptions.InvalidTokenError,
             jwt.exceptions.InvalidSignatureError,
     ):
-        return 'Invalid auth token!', 401
+        error(401, 'Invalid auth token, signature verification failed')
     except jwt.exceptions.ExpiredSignatureError:
-        return 'auth token expired!', 401
+        error(401, 'Auth token expired')
 
 class AuthToken(MethodView):
     def get(self):
```

Comment:
This is outside of this PR, but this handler should be renamed to `post()`. Any time we send data to the server the in a body the method is either `POST` or `PUT`. `GET` that sends data would use query params (and since query params are plain text we would never send sensitive data like credentials in the URL)

See [issue 320](https://github.com/codeforpdx/recordexpungPDX/issues/320).